docs/ReleaseProcess.md

```diff
@@ -137,8 +137,6 @@ releasing process), but if you are a contributor and have been asked to do this,
     PS D:\work\ebpf-for-windows> .\scripts\update-product-version.ps1 0 12 0
     Updating the version number in the 'D:\work\ebpf-for-windows\scripts\..\resource\ebpf_version.h' file...
```

Comment:
Lines 138-139 look wrong to me based on the change to line 11 in \update-product-version.ps1